resources/qml/pages/MessagingSettingsPage.qml

```diff
@@ -63,9 +63,9 @@ Page {
 
         TextSwitch {
             id: sendOmemoMsg
-            enabled: shmoose.isOmemoUser(conversationId)
+            enabled: ( shmoose.isOmemoUser(conversationId) === true || shmoose.rosterController.isGroup(conversationId) === true )
```

Comment:
Is this really enough to check for omemo on groups? As far as I know, The own jid has to be subscribed to every room members jid in order to get the room omemo encryption working. This must be checked.

Reply:
I will check

Reply:
It works fine.

Reply:
Do you have a subscription with your jid to the other two users? Are they in your roster? Please try a group chat with jid's that are not in your roster. On this group, the omemo messaging will fail, as far as I know.